e2e/app/aou-elements/web-component.ts

```diff
@@ -0,0 +1,70 @@
+import {Page} from 'puppeteer';
+import Button from './button';
+import Checkbox from './checkbox';
+import Label from './label';
+import Link from './link';
+import RadioButton from './radiobutton';
+import Select from './select';
+import TextOptions from './text-options';
+import Textarea from './textarea';
+import Textbox from './textbox';
+
+export default class WebComponent {
```

Comment:
opt naming nit (discussed in person): consider renaming this to BaseComponent, for consistency w/ the BasePage naming scheme. Even if there is a slight difference, since BasePage is absract, I think the parallel would be helpful.

Reply:
I think you meant `WebElement`, right? I've renamed it to `BaseElement`.